### PR TITLE
Initial repo configuration

### PR DESCRIPTION
**Pull request by ch-julie-kunnumpurath**
I pulled the common files and configurations in ingenuity-planner and ingenuity-codesets. Both the Application and ApplicationTest run without any errors. I also tested gitflow which seems to be working.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+	<version>0.0.1-SNAPSHOT</version>
+	<packaging>jar</packaging>
+
+	<name>ingenuity-edits</name>
+	<description>This project provides services for Ingenuity edits</description>
+
+	<parent>
+		<groupId>com.collectivehealth.ingenuity</groupId>
+		<artifactId>ingenuity-starter-parent</artifactId>
+		<version>0.0.12</version>
```
Comment (CH-EricLundberg) [inline pom.xml]: Maybe update to the current version of parent pom (unless there is some reason to stay on this one?)

File: run_integration_against_docker.sh [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```
Comment (CH-RaphaelNGouan) [inline run_integration_against_docker.sh]: We should remove this file. Not needed by jenkins

File: pom.xml [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+
+	<build>
+		<plugins>
+			<plugin>
+				<groupId>org.springframework.boot</groupId>
+				<artifactId>spring-boot-maven-plugin</artifactId>
+			</plugin>
+			<plugin>
+				<groupId>com.mysema.maven</groupId>
+				<artifactId>apt-maven-plugin</artifactId>
```
Comment (CH-RaphaelNGouan) [inline pom.xml]: Remove this plugin: not needed

**Comment by ch-julie-kunnumpurath** [comment]
Jenkins is all set up now, so this should be ready for another pass through

File: pom.xml [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+		</dependency>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-validation</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-web</artifactId>
+		</dependency>
+		<dependency>
```
Comment (CH-EricLundberg) [inline pom.xml]: Not sure we need H2 in there?

**Pull request merged**